packages/react-liveness/jest.config.js

```diff
@@ -5,10 +5,10 @@ module.exports = {
   coveragePathIgnorePatterns: ['<rootDir>/src/(index|styles|version).(ts|tsx)'],
   coverageThreshold: {
     global: {
-      branches: 65,
-      functions: 76,
-      lines: 77,
-      statements: 78,
+      branches: 80,
+      functions: 80,
+      lines: 80,
+      statements: 80,
```

Comment:
Maybe we can bump these to where the coverage is at now?

Reply:
oh doh sure that makes more sense lol